src/components/Characters/Character.js

```diff
@@ -2,27 +2,53 @@ import React, {useState} from 'react'
 import './Character.css'
 
 const Character = ({props, index}) => {
-    const {image,name,status,species,gender} = props
+    const {image,name,status,species,gender, episode} = props
```

Comment:
bad formatting, I'd recommend installing ESLint and extend from the Airbnb to make this auto-fixable once you save changes

Reply:
Sure